cmd/collaborators/update.go

```diff
@@ -92,8 +80,10 @@ func runUpdateCommand(cmd *cobra.Command, clients *shared.ClientFactory, args []
 			return err
 		}
 	} else {
-		cmd.Println(fmt.Sprintf("\n%s Specify a permission type for your collaborator with the %s flag\n", style.Emoji("warning"), style.Highlight("--permission-type")))
-		return nil
+		slackUser.PermissionType, err = promptCollaboratorPermissionSelection(ctx, clients)
+		if err != nil {
+			return err
+		}
```

Comment:
When updating an existing collaborator however, maybe it would make sense to prompt for the permission type rather than error out

Reply:
If we choose to prompt, than we should handle non-TTY (no interactivity) as well. In that case, I think we would error when there is no flag value provided.

suggestion: Personally, I suggest we keep the error when the flag is not provided because it's been tested and works. If we switch to a prompt, I'd rather see it as a separate PR because it changes the functionality rather than simply unhides the experiment (purpose of this PR).